src/modules/credentials/educationDetails/useEducationDetails.tsx

```diff
@@ -89,18 +91,24 @@ export const useEducationDetails = (
 
     const initialVal = {
       credentialName: education?.degree || '',
-      month: {
-        label: awardedDate ? monthNames[awardedDate.getMonth()] : '',
-        value: awardedDate ? awardedDate.getMonth().toString() : '',
-      },
-      day: {
-        label: awardedDate?.getDate() || '',
-        value: awardedDate?.getDate() || '',
-      },
-      year: {
-        label: awardedDate?.getFullYear() || '',
-        value: awardedDate?.getFullYear() || '',
-      },
+      month: awardedDate
+        ? {
+            label: monthNames[awardedDate.getMonth()] || '',
+            value: awardedDate.getMonth().toString() || '',
+          }
+        : null,
+      day: awardedDate
+        ? {
+            label: awardedDate?.getDate() || '',
+            value: awardedDate?.getDate() || '',
+          }
+        : null,
+      year: awardedDate
+        ? {
+            label: awardedDate?.getFullYear() || '',
+            value: awardedDate?.getFullYear() || '',
+          }
```

Comment:
getMonth() and getFullYear() return number which causes TS errors, because year and month are string.